docs/operations/node_assignment.md

```diff
@@ -140,3 +140,24 @@ An example for `tolerations` may look like this:
         operator: "Equal"
         value: "value"
         effect: "NoSchedule"
+
+## Node balancing with Descheduler
+
+In some cases we might need to rebalance the cluster on current scheduling policy 
+and load conditions. [Descheduler](https://github.com/kubernetes-sigs/descheduler) 
+can find pods, which violates e.g. scheduling decisions and evict them based on descheduler 
+policies. Kubevirt VMs are handled as pods with local storage, so by default, 
+descheduler will not evict them. But it can be easily overridden by adding special 
+annotation to the VMI template in the VM:
+
+    ```console
```

Comment:
There is a formatting problem here that you can see in the [rendered preview](https://deploy-preview-387--kubevirt-user-guide.netlify.app/operations/node_assignment/#node-balancing-with-descheduler).

Either remove the triple-quote wrapper (consistent with other yaml examples in this file) or remove the indentation of this section.